projects/common-form-elements/src/lib/dynamic-dropdown/dynamic-dropdown.component.ts

```diff
@@ -21,7 +21,7 @@ export class DynamicDropdownComponent implements OnInit, OnChanges, OnDestroy {
   @Input() isMultiple?: boolean;
   @Input() context?: FormControl;
   @Input() contextTerms?: any;
-  @Input() formControlRef?: CustomFormControl;
+  @Input() formControlRef?: FormControl;
```

Comment:
`CustomFormControl` is needed. can you please revert this change?

Reply:
Done